Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py

```diff
@@ -894,6 +943,152 @@ def reputation_operation_command(client_obj, indicator, reputation_function):
+                if info['Timestamp'] >= asset_alert_hr['Last Seen Ago']:
+                    asset_alert_hr['Last Seen Ago'] = info['Timestamp']
+                    asset_alert_hr['Last Seen'] = get_informal_time(info['Timestamp'])
+                    asset_alert_ctx['LastSeen'] = info['Timestamp']
+                elif info['Timestamp'] <= asset_alert_hr['First Seen Ago']:
+                    asset_alert_hr['First Seen Ago'] = info['Timestamp']
+                    asset_alert_hr['First Seen'] = get_informal_time(info['Timestamp'])
+                    asset_alert_ctx['FirstSeen'] = info['Timestamp']
+            else:
+                asset_alert_hr['First Seen Ago'] = info['Timestamp']
+                asset_alert_hr['First Seen'] = get_informal_time(info['Timestamp'])
+                asset_alert_hr['Last Seen Ago'] = info['Timestamp']
+                asset_alert_hr['Last Seen'] = get_informal_time(info['Timestamp'])
+
+                asset_alert_ctx['FirstSeen'] = info['Timestamp']
+                asset_alert_ctx['LastSeen'] = info['Timestamp']
+
+            asset_alert_ctx.setdefault('Occurrences', []).append(info['Timestamp'])
+            asset_alert_ctx['Alerts'] = asset_alert_hr['Alerts'] = asset_alert_ctx.get('Alerts', 0) + 1
+            asset_alert_ctx['Asset'] = asset_alert['AssetName']
+            asset_alert_ctx['AlertName'] = asset_alert_hr['Alert Names'] = info['Name']
+            asset_alert_ctx['Severities'] = asset_alert_hr['Severities'] = info['Severity']
+            asset_alert_ctx['Sources'] = asset_alert_hr['Sources'] = info['SourceProduct']
+
+            asset_alert_hr['Asset'] = '[{}]({})'.format(asset_alert['AssetName'], info.get('Uri'))
+
+            unique_asset_alert_ctx[asset_alert_key] = asset_alert_ctx
+            unique_asset_alerts_hr[asset_alert_key] = asset_alert_hr
+
+    return unique_asset_alerts_hr, unique_asset_alert_ctx
+
+
+def convert_alerts_into_hr(events):
+    """
+    converts alerts into human readable by parsing alerts
+    :param events:
+    :return:
+    """
+    data = group_infos_by_alert_asset_name(events)[0].values()
+    return tableToMarkdown('Security Alert(s)', list(data),
+                           ['Alerts', 'Asset', 'Alert Names', 'First Seen', 'Last Seen', 'Severities',
+                            'Sources'],
+                           removeNull=True)
+
+
+def get_asset_identifier_details(asset_identifier):
+    """
+    Return asset identifier detail such as hostname, ip, mac
+
+    :param asset_identifier: A dictionary that have asset information
+    :type asset_identifier: dict
+
+    :return: asset identifier name
+    :rtype: str
+    """
+    if asset_identifier.get('hostname', ''):
+        return asset_identifier.get('hostname', '')
+    if asset_identifier.get('ip', []):
+        return '\n'.join(asset_identifier.get('ip', []))
+    if asset_identifier.get('mac', []):
+        return '\n'.join(asset_identifier.get('mac', []))
+    return '-'
```

Comment:
why not None, would make this go away if you use the removeNulls

Reply:
making this change.